test/functionalTest/orionWS.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import sys
+import websocket
+
+msg = 0;
+
+def on_message(ws, message):
+    print message
+
+    global msg
+    msg -= 1
+    if (msg == 0):
+        ws.close()
+
+def on_error(ws, error):
+    print error
+
+def on_close(ws):
+    print "Connection closed"
+
+def on_open(ws):
+    ws.send(sys.argv[3])
+
+def show_help(reason):
```

Comment:
Fixed in c10db2a2e77cf4b6e8b8013c80440c26e55c7383

Need to modify the CMakeLists.txt in order to "install" orionWS.py in a directory listed in PATH variable (like accumulaor-server.py)